tools/test/qa/tests/stability_test/multiplex_pipeline-test.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+# Simple script for simultaneous multiplex pipeline testing
+
+NUMBER=$#	# number of test pipeline
+PARAMS=$@	# name of test pipeline
+MAXLOOPS=1000	# loop number
+FORMATE=s16_le	# sample formate
+CHANNEL=2	# test channel number
+FREQENCY=48000	# sample frequency
+ITERVAL=5	# iterval time for each iteration
```

Comment:
I guess you missed 'n' "iNterval" ;)
Also I think values can be smaller than 5 seconds, can it?

Reply:
Thanks for your review, indeed, this is a typo.